host-wit/deps/spin-wasi-virt/world.wit

```diff
@@ -26,4 +26,15 @@ world virt {
     export wasi:sockets/tcp@0.2.0;
     export wasi:sockets/tcp-create-socket@0.2.0;
     export wasi:sockets/ip-name-lookup@0.2.0;
+
+    /// Helpers interfaces to dynamically configure virtualization.
+    export environment;
 }
+
+/// Interface for configuring the behavior of the `wasi:cli/environment` interface
+interface environment {
+    /// Set the POSIX-style arguments to the program.
+    set-arguments: func(arguments: list<string>);
```

Comment:
hmmm... I don't believe Spin apps have any `args` available to them. I believe `args` should always be empty, and we shouldn't provide any way to set arguments.